lightning/src/util/test_utils.rs

```diff
@@ -172,13 +172,17 @@ pub struct TestPersister {
 	/// When we get an update_persisted_channel call with no ChannelMonitorUpdate, we insert the
 	/// MonitorUpdateId here.
 	pub chain_sync_monitor_persistences: Mutex<HashMap<OutPoint, HashSet<MonitorUpdateId>>>,
+	/// When we get an update_persisted_channel call *with* a ChannelMonitorUpdate, we insert the
+	/// MonitorUpdateId here.
+	pub offchain_sync_monitor_persistences: Mutex<HashMap<OutPoint, HashSet<MonitorUpdateId>>>,
```

Comment:
To me, the word "sync" doesn't make sense in this field name. The phrase "offchain sync" is a little weird to me, maybe `offchain_upd` would make more sense

Reply:
Went with `offchain_monitor_updates`